Process.c

```diff
@@ -1274,14 +1277,13 @@ void Process_updateFieldWidth(ProcessField key, size_t width) {
 }
 
 void Process_updateCPUFieldWidths(float percentage) {
-   if (percentage < 99.9) {
+   if (percentage < 99.9F) {
       Process_updateFieldWidth(PERCENT_CPU, 4);
```

Comment:
```suggestion
      Process_updateFieldWidth(PERCENT_CPU, 5);
```

The cause of the misaligned title comes from here. You missed this change from the patch I attached.

Reply:
Thank you!

Reply:
When @BenBE pushed a new commit, this width change I suggested to @etanot seems to be reverted. The current code of the PR has the minimum CPU% field width of 4.

Reply:
Based on the width of the column header it uses a minimum of 5, even though the code would allow it to be 4.

Reply:
Why should the code allow it to be 4 anyway?

Reply:
Because I'm not special-casing things that do not need special-casing …

The minimum sensible width is 4, thus a check for 99.9 is sufficient. Any column that has a larger caption inherits a wider minimum width automatically without need to do any other checks. That's also why I dropped that one commit while updating the PR.

Reply:
I thought the function name `Process_updateCPUFieldWidths` implies this is a special case anyway.

I would have been comviced if the function name were `Process_updatePercentageFieldWidth`.

Reply:
I won't mind renaming that function.